src/Microsoft.Framework.PackageManager/Algorithm/DepthFirstPostOrderWalker.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Microsoft.Framework.PackageManager.Algorithm
```

Comment:
Should reconsider this namespace. It should probably at least be plural because there are multiple algorithms here, but beyond that, just "algorithm" is awfully generic. What kind of algorithms are in here?